src/main/scala/org/holmesprocessing/totem/services/passivetotal2/src/passivetotal-service/main.go

```diff
@@ -0,0 +1,277 @@
+package main
+
+/*
+ * Imports for messuring execution time of requests
+ */
+import (
+    "time"
+)
+
+/*
+ * Imports for reading the config, logging and command line argument parsing.
+ */
+import (
+    "os"
+    "path/filepath"
+    "log"
+    "flag"
+    "fmt"
+    "passivetotal-service/config"
+)
+
+/*
+ * Imports for serving on a socket and handling of incoming request.
+ */
+import (
+    "net/http"
+    "github.com/julienschmidt/httprouter"
+    "encoding/json"
+    "io/ioutil"
+)
+
+/*
+ * Imports relevant for service execution
+ */
+import (
+    "sync"
+    "passivetotal-service/passivetotal"
+)
+
+
+// global variables
+var (
+    cfg        *Config
+    infoLogger *log.Logger
+)
+
+
+// config structs
+type Metadata struct {
+    Name                string
+    Version             string
+    Description         string
+    Copyright           string
+    License             string
+}
+type Settings struct {
+    Port                string
+    InfoURL             string
+    AnalysisURL         string
+    APIUser             string
+    APIKey              string
+    RequestTimeout      int
+}
+type Config struct {
+    Metadata            Metadata
+    Settings            Settings
+}
+
+
+// main logic
+func main () {
+    var configPath string
+
+    // setup logging
+    infoLogger = log.New(os.Stdout, "", log.Ltime|log.Lshortfile)
+
+    // load config
+    flag.StringVar(&configPath, "config", "", "Path to the configuration file")
+    flag.Parse()
+    if configPath == "" {
+        configPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
+        configPath    = filepath.Join(configPath, "service.conf")
+    }
+    cfg = &Config{}
+    config.Parse(cfg, configPath)
+
+    // modify analysis routing url
+    analysisURL := cfg.Settings.AnalysisURL
+    if analysisURL[len(analysisURL)-1] != '/' {
+        analysisURL += "/"
+    }
+    analysisURL += ":object"
+
+    // setup http handlers
+    router := httprouter.New()
+    router.GET(analysisURL, handlerAnalyze)
+    router.GET(cfg.Settings.InfoURL, handlerInfo)
```

Comment:
That is because it is not necessary for the info url, we don't have a parameter there, we need the slash to make the parameter work correctly